docs/reference/Image.rst

```diff
@@ -59,7 +59,7 @@ Functions
         documentation`_ to have warnings output to the logging facility instead of stderr.
 
 	.. _decompression bombs: https://en.wikipedia.org/wiki/Zip_bomb
-	.. _the logging documentation: https://docs.python.org/2/library/logging.html?highlight=logging#integration-with-the-warnings-module
+	.. _the logging documentation: https://docs.python.org/3/library/logging.html#integration-with-the-warnings-module
```

Comment:
What was your reason for removing '?highlight=logging'?

Reply:
It looked odd to me and was perhaps copied & pasted mistakenly. But I've added it back. The change now only updates 2 -> 3. Thanks.

Reply:
Oh, you can leave if you like, I just wanted to understand why. Thanks